src/machine/behavior/is/mod.rs

```diff
@@ -23,14 +23,17 @@ pub mod system;
 
 use machine::state::State;
 
-pub fn get(_opcode: u8) -> Semantic {
-    unimplemented!();
-    // SEMANTICS[opcode]
+pub fn get(opcode: u8) -> Semantic {
+    SEMANTICS[opcode as usize]
+}
+
+pub fn instruction_name(opcode: u8) -> &'static str {
```

Comment:
Why would we return a tuple if we don't actually need the two values at the same time?